### ISSUE TITLE
Remaining accessibility issues

### ISSUE THREAD

**Issue by alice** [Polymer/paper-button#23]
- Focus style (usual disclaimer) - perhaps we need to motivate the design team by drawing the default focus ring around everything until they figure it out :)
- `disabled` state should be reflected in `aria-disabled` state (note that you can "click" it using the screen reader without this state, triggering the "this shouldn't happen" dialog)
- Keyboard events: http://www.w3.org/TR/wai-aria-practices/#button
- `aria-label` seems to be unnecessary given the text is in descendant content, but maybe better safe than sorry

**Comment by morethanreal** [comment]
I actually agree with @alice about the default blue focus ring. The one implemented in 8cc8a41 is non-standard wrt material design and subtle compared to the default. Re-opening to turn on the default everywhere.
